NSS-Day-Cohort-49/react-nutshell-state-slingers
# Lam messages

#### Changes Made
1. Modified file `MessageList.js and MessageCard.js  and css files` to include `edit functinality, add friend from chat history functionality, sending and receiving private messages to a friend only visible to a friend and css styling accordingly`
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user clicks on edit button on a message that user sent, only the can delete and modify.

4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
